x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/DocumentAndFieldLevelSecurityTests.java

```diff
@@ -42,7 +42,8 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 
-public class DocumentAndFieldLevelSecurityTests extends SecurityIntegTestCase {
+@ESIntegTestCase.ClusterScope(numClientNodes = 1)
```

Comment:
Can we move the tests in this class to the newly introduced [DocumentLevelSecurityFeatureUsageTests.java](https://github.com/elastic/elasticsearch/pull/88708/files#diff-d201e525f0cefb113fff77d4e78fe2bbcaa4d5d77267849e0ac34c8db8c1a888) one.

This test class already tests too much, and I'm worried that if we restrict the configuration (the number of client nodes) we might miss some esotheric test condition.

Reply:
Done. I also introduced a new test class `FieldLevelSecurityFeatureUsageTests`, to avoid changing `FieldLevelSecurityTests` for the same reason.